src/Form/index.tsx

```diff
@@ -769,6 +771,24 @@ function Form({
 
           if (toAwait) await toAwait;
 
+          if (logicRule.server_side) {
+            try {
+              const response = await client.runServerSideLogicRule(
+                logicRule.id
+              );
+              if (response?.field_data) {
+                setFieldValues(response?.field_data, true, true);
+              } else if (response?.error) {
+                handleRuleError(response?.error, logicRule);
+              }
+            } catch (e: any) {
+              const errorMessage =
+                e.reason?.message ?? e.error?.message ?? e.message;
+              handleRuleError(errorMessage, logicRule);
+            }
+            continue;
```

Comment:
i think this would read clearer as an if-else rather than a continue in the if statement